python/python/glide/async_commands/server_modules/ft.py

```diff
@@ -0,0 +1,56 @@
+# Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0
+"""
+module for `vector search` commands.
+"""
+
+from typing import List, Optional, cast
+
+from glide.async_commands.server_modules.ft_constants import (
+    CommandNames,
+    FtCreateKeywords,
+)
+from glide.async_commands.server_modules.ft_options.ft_create_options import (
+    Field,
+    FtCreateOptions,
+)
+from glide.constants import TOK, TEncodable
+from glide.glide_client import TGlideClient
+
+
+async def create(
+    client: TGlideClient,
+    indexName: TEncodable,
+    schema: List[Field],
```

Comment:
I believe this should be optional to, and defaults to None, since we dont force the user to send any schema, we shouldn't force sending an empty list

Reply:
or maybe its not optional and I am having a hard time reading the aws documentation

Reply:
It is not optional. We need atleast one field on the schema to create an index.